client/src/call-customer/components/board.tsx

```diff
@@ -0,0 +1,76 @@
+import { useState, useRef } from 'react'
+import '../call-customer.css';
+import Clock from './clock.tsx';
+import { BsFillPeopleFill } from "react-icons/bs";
+import { BsFillPersonFill } from "react-icons/bs";
+import { TbHourglassEmpty } from "react-icons/tb";
+
+function DisplayBoard() {
+
+    const [ticketNo, setTicketNo] = useState(110);
+    const queue1: number = 0;
+    const queue2: number = 1;
+    const queue3: number = 5;
+
+
+    return (
+        <>
+            <div className='full-page'>
+                <div className='column'>    
+                    <Clock />     
+                    {/* Ticket Number Box */}
+                    <div className='ticket-box'>
```

Comment:
This could be a possible candicate for Separation of Concerns principle.
It means you may want to consider this as a separate component e.g BoardNextCustomerCard

Reply:
Since it's a small page, I would keep this as it is for now